jobs/payment-jobs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 -e git+https://github.com/bcgov/sbc-common-components.git@e58dc66d58f0c783ffad5452673d9d4f110cb376#egg=sbc_common_components&subdirectory=python
--e git+https://github.com/saravanpa-aot/sbc-pay.git@e38e9798dd62125856a95ee2e6b7253a5fd8beae#egg=pay_api&subdirectory=pay-api
+-e git+https://github.com/sumesh-aot/sbc-pay.git@40b1b1764473931e5660e2f7e0a8907150bbe69c#egg=pay_api&subdirectory=pay-api
```

Comment:
should we use bcgov link here?

Reply:
The changes needed for the payment-jobs are in this commit under pay-api, so build wont work. I will create a PR once this is merged